src/lib/DynamoDB.js

```diff
@@ -0,0 +1,137 @@
+import { 
+  BatchWriteItemCommand,
+  CreateTableCommand, 
+  DeleteTableCommand, 
+  DescribeTableCommand, 
+  DynamoDBClient, 
+  GetItemCommand, 
+  ListTablesCommand, 
+  QueryCommand, 
+  ScanCommand, 
+  UpdateTableCommand,
+} from "@aws-sdk/client-dynamodb";
+import { 
+  DeleteCommand, 
+  DynamoDBDocumentClient, 
+  PutCommand, 
+  UpdateCommand, 
+} from '@aws-sdk/lib-dynamodb';
+  
+const DYNAMO_DB_AWS_ACCESS_KEY =
+    process.env.KS_DYNAMO_DB_AWS_ACCESS_KEY_ID ??
+    process.env.KS_AWS_ACCESS_KEY_ID ??
+    '';
+const DYNAMO_DB_AWS_SECRET_ACCESS_KEY =
+    process.env.KS_DYNAMO_DB_AWS_SECRET_ACCESS_KEY ??
+    process.env.KS_AWS_SECRET_ACCESS_KEY ??
+    '';
+const DYNAMO_DB_AWS_REGION =
+    process.env.KS_DYNAMO_DB_AWS_REGION ??
+    process.env.KS_AWS_REGION ??
+    'us-east-1';
+  
+const defaultMarshallOptions = {
+    convertEmptyValues: false, // Whether to automatically convert empty strings, blobs, and sets to `null`. false, by default. 
+    removeUndefinedValues: true, // Whether to remove undefined values while marshalling. false, by default.
+    convertClassInstanceToMap: false, // Whether to convert typeof object to map attribute. false, by default.
+};
+      
+const defaultUnmarshallOptions = {
+    wrapNumbers: false, // Whether to return numbers as a string instead of converting them to native JavaScript numbers. false, by default.
+};
+  
+class DynamoDB {
+    constructor({ marshallOptions, unmarshallOptions } = {}) {
+      const DDBConfig = {
+        region: DYNAMO_DB_AWS_REGION,
+        credentials: {
+          accessKeyId: DYNAMO_DB_AWS_ACCESS_KEY,
+          secretAccessKey: DYNAMO_DB_AWS_SECRET_ACCESS_KEY,
+        }
+      }
+    
+      const DDBDocConfig = {
+        marshallOptions: marshallOptions ?? defaultMarshallOptions,
+        unmarshallOptions: unmarshallOptions ?? defaultUnmarshallOptions,
```

Comment:
Allow partial override, so if the developer specified `convertEmptyValues` and nothing else, then it will still inherit the fields where the developer failed to explicitly define.
```suggestion
        marshallOptions: {...defaultMarshallOptions, ...marshallOptions},
        unmarshallOptions: {...defaultUnmarshallOptions, ...unmarshallOptions},
```